src/virgil-crypto.js

```diff
@@ -166,9 +166,9 @@ export const virgilCrypto = {
     ));
   },
 
-  extractPublicKey(virgilPrivateKey) {
-    const privateKeyValue = checkedGetPrivateKeyValue(virgilPrivateKey);
-    const { identifier, publicKey } = unwrapResponse(RNVirgilCrypto.extractPublicKey(privateKeyValue));
+  extractPublicKey(virgilPublicKey) {
+    const publicKeyValue = checkedGetPublicKeyValue(virgilPublicKey);
+    const { identifier, publicKey } = unwrapResponse(RNVirgilCrypto.extractPublicKey(publicKeyValue));
     return new VirgilPublicKey(identifier, publicKey);
   },
```

Comment:
Thanks for your update.
Making this change will result in an error.
I don't understand the details, but I think the original code is correct since it seems to be getting the public key from the private key.

Reply:
@tmizu23 Hello! Ty for reporting, fix in new 0.6.3 version!